x-pack/plugins/cloud_security_posture/public/components/no_vulnerabilities_states.tsx

```diff
@@ -59,14 +58,14 @@ const ScanningVulnerabilitiesEmptyPrompt = () => (
   />
 );
 
-const VulnerabilitiesFindingsInstalledEmptyPrompt = ({
+const VulnerabilitiesVulnerabilitiesInstalledEmptyPrompt = ({
```

Comment:
Why `Vulnerabilities` twice?

also I would add a "not" before the installed
```suggestion
const CnvmIntegrationNotInstalledEmptyPrompt = ({
```

Reply:
This should be `Vulnerabilities` used once here. Good catch.